config/core/test-configs.yaml

```diff
@@ -2078,15 +2095,24 @@ test_configs:
     test_plans:
       - baseline_qemu
       - baseline-qemu-docker
+      - kselftest-filesystems
+      - kselftest-futex
+      - kselftest-lib
```

Comment:
A template is missing for running kselftest on QEMU, see this job error:
```
Template not found: config/lava/kselftest/generic-qemu-tftp-nfs-kselftest-template.jinja2
```
One issue here is that it's going to be a bit complicated to specify a QEMU template name for each kselftest collection.  So it might be better to drop QEMU from this PR and make another one just for that, so we can merge this one more easily.

Reply:
ok, qemu removed from this PR. I will open a new soon :)